elements/map/src/helpers/draw.js

```diff
@@ -77,3 +140,35 @@ export function addDraw(EOxMap, drawLayer, options) {
   // Subscribe to the 'change' event on the layer group to detect when layers are removed
   EOxMap.map.getLayerGroup().on("change", removeLayerListener);
 }
+
+/**
+ * Calculate real distance on the map and format the output.
+ * @param {Polygon} line The line string to calculate the distance for.
+ * @return {string} Formatted length.
+ */
+const formatLength = function (line) {
+  const length = getLength(line);
+  let output;
+  if (length > 100) {
+    output = Math.round((length / 1000) * 100) / 100 + " " + "km";
+  } else {
+    output = Math.round(length * 100) / 100 + " " + "m";
+  }
+  return output;
+};
+
+/**
+ * Format area output.
+ * @param {Polygon} polygon The polygon.
+ * @return {string} Formatted area.
+ */
+const _formatArea = function (polygon) {
```

Comment:
Is this used anywhere? It would be nice to support both lenght as well as area measurement.

Reply:
Yes, as currently implemented we would add it as a feature of the map, since that is where the OL interaction is created.

Currently this is not used, since the goal is mostly to determine distances in the GTIF apps. Question with two-dimensional measurements is also how the tooltip should behave (properly centered or stationary in a corner of the map).